File: snippets/api/activations/register-is-icon.java
PlasmoVoiceServer voiceServer = /* */;
Object addon = /* */;

try (InputStream is = Files.newInputStream(new File("icon.png").toPath())) {
    ServerActivation activation = voiceServer.getActivationManager().createBuilder(
        addon,
        "priority", // name
        "pv.activation.priority", // translation key
        is, // icon
        "pv.activation.priority", // permission
        10 // weight
    ).build();
} catch (IOException e) {
    e.printStackTrace();
}

File: snippets/api/activations/register-is-icon.kt
val voiceServer: PlasmoVoiceServer
val addon: Any

val activation = Files.newInputStream(File("icon.png").toPath()).use { icon ->
    voiceServer.activationManager.createBuilder(
        this,
        "priority", // name
        "pv.activation.priority", // translation key
        icon, // icon
        "pv.activation.priority", // permission
        10 // weight
    ).build()
}
